src/pages/EditMonitor.vue

```diff
@@ -49,8 +49,9 @@
                     </div>
 
                     <div class="mb-3">
-                        <label for="maxRetries" class="form-label">Retry Pings</label>
+                        <label for="maxRetries" class="form-label">Retries</label>
                         <input type="number" class="form-control" id="maxRetries" v-model="monitor.maxretries" required min="0" step="1">
+                        <div class="form-text">Maximum retries before the service is marked as down and send notifications</div>
```

Comment:
I believe
`"Maximum retries before the service is marked as down and a notification is sent"`
would be more grammatically correct.
Any thoughts?